test-packages/e2e-tests/test/request-builder.spec.ts

```diff
@@ -98,6 +103,76 @@ describe('Request builder test', () => {
     );
   });
 
+  it('should create an entity with related entities (deep create)', async () => {
```

Comment:
So, this is THE activated test for deep create.

Reply:
yes

Reply:
more was not supported by the services, but at least something.